common/types/chat_saved_object_attributes.ts

```diff
@@ -43,11 +43,11 @@ export interface IOutput {
   type: 'output';
   traceId?: string; // used for tracing agent calls
   toolsUsed?: string[];
-  // TODO: ppl_visualization type may need to be removed in the PR which replaces ppl query render from visualization to data grid. @suzhou
-  contentType: 'error' | 'markdown' | 'visualization' | 'ppl_visualization' | 'ppl_data_grid';
+  contentType: 'error' | 'markdown' | 'visualization' | 'ppl_data_grid' | string;
```

Comment:
Shall we use `string` directly?

Reply:
I'd like to keep some default enum here to indicate these are built-in contentTypes, but in that case maybe we should remove `ppl_data_grid`?